tests/tracing/test_integration_tests.py

```diff
@@ -1,6 +1,5 @@
 import weakref
 import gc
-
```

Comment:
I think in pep8 the recommendation is to keep a blank line between stdlib imports and 3rd party imports.  Not sure why this is removed?